src/player-actions/violence/types.ts

```diff
@@ -56,16 +53,29 @@ export interface AlterMomentumAction {
   momentum: number;
 }
 
-export const SELECT_COMBAT_ACTION = "SELECT_COMBAT_ACTION";
-export interface SelectCombatActionAction {
-  type: typeof SELECT_COMBAT_ACTION;
+export const SELECT_PLAYER_COMBAT_ACTION = "SELECT_PLAYER_COMBAT_ACTION";
+export interface SelectPlayerCombatActionAction {
+  type: typeof SELECT_PLAYER_COMBAT_ACTION;
+  combatAction: CombatAction;
+}
```

Comment:
This is all noisy boilerplate that can all be reduced using some convention or library (e.g. redux-toolkit)